model.go

```diff
@@ -162,6 +162,16 @@ func (n *Node) Summary() string {
 	} else if len(n.Children) > 0 {
 		summary += " <command>"
 	}
+	allFlags := n.Flags
+	if n.Parent != nil {
+		allFlags = append(allFlags, n.Parent.Flags...)
+	}
+	for _, flag := range allFlags {
+		if !flag.Required {
+			summary += " [optional flags]"
```

Comment:
Make this just `[<flags>]` for consistency - the `[]` already implies optional, and `<flags>` is consistent with eg. `<command>` and `<arg>`.

Reply:
Done